.travis.yml

```diff
@@ -0,0 +1,22 @@
+# Config file for automatic testing at travis-ci.org
+
+sudo: false
+
+language: python
+
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 2.7
+      env: TOXENV=quality
+  allow_failures:
+    - TOXENV=py35
+
+install:
+  - pip install -r requirements.txt
+
+script:
+- tox
```

Comment:
Do you want to add codecov and deployment support now? https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/.travis.yml